fairworkflows/fairworkflow.py

```diff
@@ -59,9 +59,16 @@ def add(self, step:FairStep, follows:FairStep=None):
 
     def __iter__(self):
         """
-            Returns an iterator over this FairWorkflow, that returns one step at a time in the order specified by the precedes relations.
+        Iterate over this FairWorkflow, return one step at a
+        time in the order specified by the precedes relations (
+        i.e. topologically sorted).
         """
-        return PlexIterator(self)
+        G = nx.MultiDiGraph()
```

Comment:
This is a much cleaner implementation, thanks

Reply:
Great, thanks!